src/QsCompiler/Core/SymbolTable.fs

```diff
@@ -1342,16 +1427,16 @@ and NamespaceManager
     member this.NamespacesContainingCallable cName = 
```

Comment:
It would be good to mention in the doc comment whether private/internal things are included.